tst/EnergyPlus/unit/SurfaceGeometry.unit.cc

```diff
@@ -7021,3 +7021,532 @@ TEST_F(EnergyPlusFixture, GetSurfaceData_SurfaceOrder)
+        "    04Wall_N,                !- Name",
+        "    Wall,                    !- Surface Type",
+        "    R13WALL,                 !- Construction Name",
+        "    ZONE ONE,                !- Zone Name",
+        "    Outdoors,                !- Outside Boundary Condition",
+        "    ,                        !- Outside Boundary Condition Object",
+        "    SunExposed,              !- Sun Exposure",
+        "    WindExposed,             !- Wind Exposure",
+        "    0.5000000,               !- View Factor to Ground",
+        "    6,                       !- Number of Vertices",
+        "    8, 4, 2.5,                          !- X,Y,Z  1 {m}",
+        "    8, 4, 0,                            !- X,Y,Z  2 {m}",
+        "    0, 4, 0,                            !- X,Y,Z  3 {m}",
+        "    0, 4, 2.5,                          !- X,Y,Z  4 {m}",
+        "    2, 4, 4,                         !- X,Y,Z  5 {m}",
+        "    6, 4, 4;                         !- X,Y,Z  6 {m}",
+
+        "BuildingSurface:Detailed,",
+        "    05Roof_C,                !- Name",
+        "    Roof,                    !- Surface Type",
+        "    ROOF31,                  !- Construction Name",
+        "    ZONE ONE,                !- Zone Name",
+        "    Outdoors,                !- Outside Boundary Condition",
+        "    ,                        !- Outside Boundary Condition Object",
+        "    SunExposed,              !- Sun Exposure",
+        "    WindExposed,             !- Wind Exposure",
+        "    0,                       !- View Factor to Ground",
+        "    4,                       !- Number of Vertices",
+        "    2, 4, 4,                         !- X,Y,Z  1 {m}",
+        "    2, 0, 4,                         !- X,Y,Z  2 {m}",
+        "    6, 0, 4,                         !- X,Y,Z  3 {m}",
+        "    6, 4, 4;                         !- X,Y,Z  4 {m}",
+
+        "BuildingSurface:Detailed,",
+        "06Roof_E,                !- Name",
+        "Roof,                    !- Surface Type",
+        "R13WALL,                 !- Construction Name",
+        "ZONE ONE,                !- Zone Name",
+        "Outdoors,                !- Outside Boundary Condition",
+        ",                        !- Outside Boundary Condition Object",
+        "SunExposed,              !- Sun Exposure",
+        "WindExposed,             !- Wind Exposure",
+        "autocalculate,           !- View Factor to Ground",
+        "4,                       !- Number of Vertices",
+        "6, 0, 4,                         !- X,Y,Z  1 {m}",
+        "8, 0, 2.5,                          !- X,Y,Z  2 {m}",
+        "8, 4, 2.5,                          !- X,Y,Z  3 {m}",
+        "6, 4, 4;                         !- X,Y,Z  4 {m}",
+
+        "BuildingSurface:Detailed,",
+        "07Roof_W,                !- Name",
+        "Roof,                    !- Surface Type",
+        "R13WALL,                 !- Construction Name",
```

Comment:
Same here.

Reply:
Same as above--construction does not matter here.